src/test/java/sirius/web/data/ExcelExportSpec.groovy

```diff
@@ -56,4 +58,38 @@ class ExcelExportSpec extends BaseSpecification {
         cleanup:
         Files.delete(testFile)
     }
+
+    def "only allow 1 million in a excel sheet rows"() {
```

Comment:
scope = Nightly - this sounds time consuming and memory hungry

Reply:
ok, but are there nightly builds for sirius-web and does anybody ever check the results of those?

Reply:
this test completes in about 25 seconds btw

Reply:
yes it does. yes they are reported. 25s = doubles the time of the current build.